### PR TITLE
[Feat]: Added avatar image in the navbar

### LINKED ISSUE #25: [Feature Request]: add avatar image in navbar
### Is your feature request related to a problem? Please describe.

image

Currently, the navbar only shows the name of the currently logged-in user which is incomplete.

### Describe the solution you'd like

image

Add the Avatar image.

### Describe alternatives you've considered

_No response_

### Developer Help

https://mui.com/material-ui/react-avatar/

Use the MUI avatars for this.

### PR DESCRIPTION
**Pull request by Doraemon012**
### Changes made:

- fix #25 
- Now, the users can see their profile image in the navbar when they are logged in.

### CODE REVIEW

**Comment by mittalvansh** [comment]
LGTM!

File: frontend/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "@mui/icons-material": "^5.11.16",
     "@mui/material": "^5.13.2",
     "axios": "^1.4.0",
+    "husky": "^8.0.3",
```
Comment (prajjwalkapoor) [inline frontend/package.json:24]: husky should be present in the dev dependency.

File: backend/package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "@nestjs/platform-express": "^9.0.0",
     "@prisma/client": "^4.14.1",
     "axios": "^1.4.0",
+    "husky": "^8.0.3",
```
Comment (prajjwalkapoor) [inline backend/package.json:36]: husky should be present in the dev dependency.

**Review by prajjwalkapoor — CHANGES_REQUESTED** [review]

**Comment by Doraemon012** [comment]
Now, I have moved husky to package.json

**Pull request closed without merging**